capabilities/src/execute_task_solution_capability.cpp

```diff
@@ -87,6 +87,8 @@ namespace move_group {
 ExecuteTaskSolutionCapability::ExecuteTaskSolutionCapability() : MoveGroupCapability("ExecuteTaskSolution") {}
 
 void ExecuteTaskSolutionCapability::initialize() {
+	action_callback_group_ =
+	    context_->moveit_cpp_->getNode()->create_callback_group(rclcpp::CallbackGroupType::Reentrant);
```

Comment:
What's the reason to use `Reentrant` here? This seems inherently unsafe if not handled appropriately.

Reply:
To make it possible to call the cancel callback while executing the goal callback